build-system/pr-check/build-targets.js

```diff
@@ -232,7 +232,15 @@ const targetMatchers = {
     );
   },
   [Targets.LINT_RULES]: (file) => {
-    return file.endsWith('.eslintrc.js') || file == 'package.json';
+    if (isOwnersFile(file)) {
+      return false;
+    }
+    return (
+      file.startsWith('build-system/eslint-rules') ||
+      file.endsWith('.eslintrc.js') ||
```

Comment:
+ .prettierrc

Reply:
Good catch. And on the same lines, also `.eslintignore` and `.prettierignore`, since changes to these files can affect linting results.